helper/certutil/types.go

```diff
@@ -251,12 +281,28 @@ func (p *ParsedCertBundle) getSigner() (crypto.Signer, error) {
 }
 
 // SetParsedPrivateKey sets the private key parameters on the bundle
-func (p *ParsedCertBundle) SetParsedPrivateKey(privateKey crypto.Signer, privateKeyType int, privateKeyBytes []byte) {
+func (p *ParsedCertBundle) SetParsedPrivateKey(privateKey crypto.Signer, privateKeyType PrivateKeyType, privateKeyBytes []byte) {
 	p.PrivateKey = privateKey
-	p.PrivateKeyType = privateKeyType
+	p.PrivateKeyType = PrivateKeyType(privateKeyType)
```

Comment:
Hmm, I guess not.  At one point I think the compiler was complaining, but I think I later changed the type in the signature. I'll pull that cast out.